pkgs/top-level/aliases.nix

```diff
@@ -50,6 +50,7 @@ mapAliases ({
   bar-xft = lemonbar-xft;  # added 2015-01-16
   bashCompletion = bash-completion; # Added 2016-09-28
   batti = throw "batti has been removed from nixpkgs, as it was unmaintained"; # added 2019-12-10
+  bazaar = throw "bazaar has been deprecated by breezy."; # added 2020-04-19;
```

Comment:
a superfluous semicolon at the end of the line

Reply:
Thank you, fixed.